### ISSUE TITLE
Feature: Add EngageBay to Form Action

### ISSUE THREAD

**Issue by Webnovo** [elementor/elementor#10046]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
It would be useful to include the EngageBay platform (similar to ActiveCampaign) among the actions of the form. This is to allow me to use Elementor's native forms without going through third parties (to date I use Gravity Form because it's the only one that offers EngageBay integration).

**Describe the solution you'd like**
Simply the possibility to enter the EngageBay KEY API and have the possibility to send the form data directly to their CRM.

**Describe alternatives you've considered**
As mentioned, I currently use Gravity Form. But I would be happy to use Elementor forms.

**Additional context**
Nothing

**Comment by dlinstedt** [comment]
I completely agree!  EngageBay is an awesome CRM, I just wish they could build better integration with Elementor, especially with Elementors' Brand New Form Builder release.

**Comment by debihazelden** [comment]
+1 for this
